stdlib/public/core/RangeReplaceableCollection.swift.gyb

```diff
@@ -373,11 +374,11 @@ public protocol RangeReplaceableCollection
     where S.Iterator.Element == Iterator.Element
   */
 
-  /// Adds the elements of a sequence to the end of the collection.
+  /// Adds the elements of a sequence or collection to the end of this
```

Comment:
Collection refines Sequence, so 'or collection' does not add any extra information, IMO.